### PR TITLE
Update webSocket BBEs

### PR DESCRIPTION
**Pull request by kalaiyarasiganeshalingam**
## Purpose
$Subject

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: examples/http-to-websocket-upgrade/http_to_websocket_upgrade.bal [hunk 1]
```diff
@@ -31,7 +31,7 @@ service httpService on new http:Listener(9090) {
     }
 
     // This is an HTTP to WebSocket upgrade resource. This is defined using the WebSocket upgrade resource config.
-    // Here you have access to the `http:Request` and to the query and path params where applicable.
+    // Here you have access to the [http:Request](https://ballerina.io/learn/api-docs/ballerina/http/objects/Request.html) and to the query and path params where applicable.
```
Comment (praneesha) [inline examples/http-to-websocket-upgrade/http_to_websocket_upgrade.bal:34]:
```suggestion
    // Here you have access to the [http:Request](https://ballerina.io/learn/api-docs/ballerina/http/objects/Request.html), query, and path params where applicable.
```

File: examples/http-to-websocket-upgrade/http_to_websocket_upgrade.bal [hunk 2]
```diff
@@ -46,7 +46,7 @@ service httpService on new http:Listener(9090) {
 
 // Note: When a WebSocket upgrade path is defined in HTTP resource configuration. <br>
```
Comment (praneesha) [inline examples/http-to-websocket-upgrade/http_to_websocket_upgrade.bal:47]:
```suggestion
// Note: When a WebSocket upgrade path is defined in the HTTP resource configuration. <br>
```

File: examples/websocket-failover/websocket_failover.bal [hunk 3]
```diff
@@ -33,7 +33,7 @@ service failoverProxyService on new http:Listener(9090) {
         wsClientEp.setAttribute(ASSOCIATED_CONNECTION, caller);
         caller.setAttribute(ASSOCIATED_CONNECTION, wsClientEp);
 
-        // Once the client is ready to receive frames the remote `ready` function
+        // Once the client is ready to receive frames the remote [ready](https://ballerina.io/learn/api-docs/ballerina/http/clients/WebSocketFailoverClient.html#ready) function
```
Comment (praneesha) [inline examples/websocket-failover/websocket_failover.bal:36]:
```suggestion
        // Once the client is ready to receive frames, the remote [ready](https://ballerina.io/learn/api-docs/ballerina/http/clients/WebSocketFailoverClient.html#ready) function
```

File: examples/websocket-proxy-server/websocket_proxy_server.bal [hunk 4]
```diff
@@ -28,7 +28,7 @@ service SimpleProxyService on new http:Listener(9090) {
         wsClientEp.setAttribute(ASSOCIATED_CONNECTION, caller);
         caller.setAttribute(ASSOCIATED_CONNECTION, wsClientEp);
 
-        // Once the client is ready to receive frames the remote function `ready`
+        // Once the client is ready to receive frames the remote function [ready](https://ballerina.io/learn/api-docs/ballerina/http/clients/WebSocketClient.html#ready)
```
Comment (praneesha) [inline examples/websocket-proxy-server/websocket_proxy_server.bal:31]:
```suggestion
        // Once the client is ready to receive frames, the remote function [ready](https://ballerina.io/learn/api-docs/ballerina/http/clients/WebSocketClient.html#ready)
```

**Comment by praneesha** [comment]
Need to add the link to the API Docs module in the .description file of the BBE.

**Comment by kalaiyarasiganeshalingam** [comment]
> Need to add the link to the API Docs module in the .description file of the BBE.

This description files already have the link.

File: examples/websocket-client/websocket_client.metatags [hunk 5]
```diff
@@ -0,0 +1,2 @@
+description: This sample demonstrates how WebSocket client can be used to connect to and interact with a WebSocket server using Ballerina.
```
Comment (Samallama) [inline examples/websocket-client/websocket_client.metatags:1]: Edit: This BBE demonstrates how a WebSocket client can be used to connect to and interact with a WebSocket server using Ballerina.

File: examples/websocket-failover/websocket_failover.metatags [hunk 6]
```diff
@@ -0,0 +1,2 @@
+description: This sample demonstrates how WebSocket failover client connects with the remaining backends when the client looses the connection.
```
Comment (Samallama) [inline examples/websocket-failover/websocket_failover.metatags:1]: Edit: This BBE demonstrates how a WebSocket failover client connects with the remaining backends when the client loses the connection.

File: examples/websocket-retry/websocket_retry.metatags [hunk 7]
```diff
@@ -0,0 +1,2 @@
+description: This sample demonstrates how WebSocket client reconnects to the same backend when the client looses the connection.
```
Comment (Samallama) [inline examples/websocket-retry/websocket_retry.metatags:1]: Edit: This BBE demonstrates how a WebSocket client reconnects to the same backend when the client loses the connection.

File: examples/websocket-proxy-server/websocket_proxy_server.metatags [hunk 8]
```diff
@@ -0,0 +1,2 @@
+description: This example demonstrates how Ballerina acts as a proxy to another remote service via WebSocket.
```
Comment (Samallama) [inline examples/websocket-proxy-server/websocket_proxy_server.metatags:1]: Edit: This BBE demonstrates how Ballerina acts as a proxy to another remote service via WebSocket.

File: examples/websocket-chat-application/websocket_chat_application.metatags [hunk 9]
```diff
@@ -0,0 +1,2 @@
+description: This sample demonstrates a simple WebSocket chat application using Ballerina.
```
Comment (Samallama) [inline examples/websocket-chat-application/websocket_chat_application.metatags:1]: Edit: This BBE demonstrates a simple WebSocket chat application using Ballerina.

File: examples/websocket-basic-sample/websocket_basic_sample.metatags [hunk 10]
```diff
@@ -0,0 +1,2 @@
+description: This sample demonstrates the basic functions of a WebSocket server.
```
Comment (Samallama) [inline examples/websocket-basic-sample/websocket_basic_sample.metatags:1]: Edit: This BBE demonstrates the basic functions of a WebSocket server.

File: examples/http-to-websocket-upgrade/http_to_websocket_upgrade.metatags [hunk 11]
```diff
@@ -0,0 +1,2 @@
+description: This sample demonstrates how an HTTP endpoint is updated to a WebSocket endpoint using an upgrade resource.
```
Comment (Samallama) [inline examples/http-to-websocket-upgrade/http_to_websocket_upgrade.metatags:1]: Edit: This BBE demonstrates how an HTTP endpoint is updated to a WebSocket endpoint using an upgrade resource.

File: examples/websocket-failover/websocket_failover.bal [hunk 12]
```diff
@@ -33,8 +33,7 @@ service failoverProxyService on new http:Listener(9090) {
         wsClientEp.setAttribute(ASSOCIATED_CONNECTION, caller);
         caller.setAttribute(ASSOCIATED_CONNECTION, wsClientEp);
 
-        // Once the client is ready to receive frames the remote `ready` function
-        // of the client needs to be called separately.
+        // Once the client is ready to receive frames, the remote [ready](https://ballerina.io/learn/api-docs/ballerina/http/clients/WebSocketFailoverClient.html#ready) function        // of the client needs to be called separately.
```
Comment (chamil321) [inline examples/websocket-failover/websocket_failover.bal:36]:
```suggestion
        // Once the client is ready to receive frames, the remote [ready](https://ballerina.io/learn/api-docs/ballerina/http/clients/WebSocketFailoverClient.html#ready) function of the client needs to be called separately.
```

File: examples/http-to-websocket-upgrade/http_to_websocket_upgrade.metatags [hunk 13]
```diff
@@ -0,0 +1,2 @@
+description: This BBE demonstrates how an HTTP endpoint is updated to a WebSocket endpoint using an upgrade resource.
+keywords: ballerina, ballerina by example, bbe, http, websocket upgrade
```
Comment (daneshk) [inline examples/http-to-websocket-upgrade/http_to_websocket_upgrade.metatags:2]:
```suggestion
keywords: ballerina, ballerina by example, bbe, http, websocket, websocket upgrade
```

**Pull request merged**